<a href="https://colab.research.google.com/github/PatrikKM/testovaci/blob/main/Write_the_code_of_a_simple_application_in_Python.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Sure, here is a simple application in Python:

In [2]:
def factorial(n):
  if n == 0:
    return 1
  else:
    return n * factorial(n - 1)

def main():
  print(factorial(5))

if __name__ == "__main__":
  main()

120


This application defines a function called `factorial()` that calculates the factorial of a number. The factorial of a number is the product of all the positive integers less than or equal to that number. For example, the factorial of 5 is 120, because 120 = 1 * 2 * 3 * 4 * 5.

The `factorial()` function uses recursion to calculate the factorial of a number. Recursion is a technique where a function calls itself to solve a problem. In this case, the `factorial()` function calls itself to calculate the factorial of the number minus 1. The base case of the recursion is when the number is 0, in which case the function simply returns 1.

The `main()` function in the application calls the `factorial()` function with the number 5 as an argument. The `factorial()` function then calculates the factorial of 5 and returns the result, which is 120.

To run this application, you can save the code above as a Python file and then run the file from the command line. For example, if you save the code as `factorial.py`, you can run the application by running the following command:

```
python factorial.py
```

This will start the application and print the factorial of 5, which is 120.

# Nová sekce